### ISSUE TITLE
Auto height and vertical submenus

### ISSUE THREAD

**Issue by Nonag** [FrDH/jQuery.mmenu#720]
Hello everyone,

the auto height option works just fine for panels that slide in. Depening on their content, the height is calculated and adapted correctly.

However, if one of these slides contains a submenu that has the class "Vertical", the height of the hidden vertical submenu is already taken into account when the panel containing it, slides in.

Is there an option im missing, to exclude the hidden vertical submenus height, when calculating the panels height?
If not, is there a chance you will be "fixing" this?

Kind regards,
Matthias
![styleguide - google chrome 2017-04-07 14 45 52](https://cloud.githubusercontent.com/assets/7794077/24800590/f0593c34-1ba0-11e7-8a0f-ce02e317893a.png)
![styleguide - google chrome 2017-04-07 14 43 30](https://cloud.githubusercontent.com/assets/7794077/24800528/bf139c00-1ba0-11e7-81de-d4dcdb4e3de9.png)

**Comment by FrDH** [comment]
Thanks for reporting this issue.

Looking at the CSS, all panels will be made visible to measure the height:
````css
.mm-menu.mm-measureheight .mm-panel
{
    display: block !important;
}
````
This should probably be expanded to exclude closed vertical panels.
I'll put it on the todo list.

**Comment by FrDH** [comment]
Should be fixed in version 6.1

**Comment by Nonag** [comment]
Just updated and tested the vertical panels. Works like a charm - much appreciated!
